apps/src/aichat/views/presentation/PresentationView.tsx

```diff
@@ -17,8 +17,11 @@ const PresentationView: React.FunctionComponent = () => {
   const {selectedModelId, systemPrompt, temperature, retrievalContexts} =
     savedAiCustomizations;
   const modelCardInfo = savedAiCustomizations.modelCardInfo;
-  const {name: modelName = '', trainingData = ''} =
-    modelDescriptions.find(model => model.id === selectedModelId) ?? {};
+  const {
+    name: modelName = '',
+    trainingData = '',
+    overview = '',
+  } = modelDescriptions.find(model => model.id === selectedModelId) ?? {};
```

Comment:
Could you just provide the defaults instead of the empty object after the `??` and then not need to default to an empty string for each? Something like
```
const { modelName, trainingData, overview } = 
   modelDescriptions.find(model => model.id === selectedModelId) ?? 
   { modelName: '', trainingData: '', overview: ''}
```